full-node/sov-sequencer/src/utils.rs

```diff
@@ -38,6 +38,39 @@ impl SimpleClient {
         Ok(())
     }
 
+    /// Sends multiple transactions to the sequencer for immediate publication.
+    pub async fn send_transactions<Tx: BorshSerialize>(
+        &self,
+        txs: Vec<Tx>,
+        micro_batch: Option<usize>,
```

Comment:
nit: The name `micro_batch` doesn't convey that this is the chunk size very clearly. Maybe it would be clearer to call it `chunk_size`? 

We could also look at changing the API to `fn send_batches(&self, batches: Vec<Vec<Tx>>` and then we wouldn't need the extra param and logic here

Reply:
Update: We never use `micro_batch`. Can we just remove it?

Reply:
I use the second parameter primarily while switching between MockDA and celestia.
switched the logic to use Vec<Vec<Tx>> so that the chunking can be handled by the caller